MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -266,6 +267,13 @@ public void SetAvailableDecoders(IEnumerable<string> list)
             //_logger.Info("Supported decoders: {0}", string.Join(",", list.ToArray()));
         }
 
+        private List<string> _hwaccels = new List<string>();
```

Comment:
```suggestion
        private List<string> _hwaccels = Array.Empty<string>;
```
or something similar?